test/interactiveutil.jl

```diff
@@ -0,0 +1,27 @@
+
+file, line = functionloc(workspace)
+func, str = editcmd(file, line)
+@test func == run || func == spawn
+@test contains(string(str), file)
+
+
+@unix_only begin
+    try
+        clipboard("Clipboard Test")
+        @test clipboard() == "Clipboard Test"
+    catch exc
+        @test isa(exc, ErrorException)
+        @test contains(exc.msg, "no clipboard command found, please install xsel or xclip")
+    end
+end
+
+# Not sure how detailed the versioninfo check needs to be
+versioninfo(true)
```

Comment:
should redirect the output to avoid printing to screen during normal operation of the tests, wherever possible